guides/common/modules/proc_synchronizing-custom-python-repositories.adoc

```diff
@@ -0,0 +1,37 @@
+[id="Synchronizing_{custompythonreposid}_{context}"]
+= Synchronizing {custompythonrepostitle}
+
+On {Project}, you can synchronize your {custompythonrepos} from any mirror and other {Project} instances.
+You can view the python packages in the {ProjectWebUI} at *Content* > *Other Content Types*.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Select a {customproduct} by name.
+. In the *Products* window, select the name of a product that you want to create a repository for.
+. Click the *Repositories* tab, and then click *New Repository*.
+. In the *Name* field, enter a name for the repository.
++
+The *Label* field is populated automatically based on the name.
+. From the *Type* list, select *python*.
+. In the *Upstream URL* field, enter the URL for the upstream content source, for example `\https://pypi.org`.
+. Optional: In the *Upstream Username* field, enter the user name for the upstream repository if required for authentication.
+Clear this field if the repository does not require authentication.
+. Optional: In the *Upstream Password* field, enter the corresponding password for the upstream repository.
+Clear this field if the repository does not require authentication.
+. Optional: In the *Upstream Authentication Token* field, provide the token of the upstream repository user for authentication.
+Leave this field empty if the repository does not require authentication.
+. Optional: In the *Excludes* field, enter a list of python packages separated by newlines to exclude from synchronizing to {Project}.
+. Optional: In the *Includes* field, enter a list of python packages separated by newlines to limit the synchronized packages.
```

Comment:
```suggestion
. Optional: In the *Includes* field, enter a list of python packages separated by newlines to limit the synchronization only to the included packages.
```
To be more specific, if this is the case.

Reply:
Applied.